Demo/src/main/java/com/braintreepayments/demo/PaymentButtonsFragment.java

```diff
@@ -10,23 +10,26 @@
 
 import com.braintreepayments.api.uicomponents.PayPalButton;
 import com.braintreepayments.api.uicomponents.PayPalButtonColor;
+import com.braintreepayments.api.uicomponents.VenmoButton;
+import com.braintreepayments.api.uicomponents.VenmoButtonColor;
 import com.google.android.material.button.MaterialButtonToggleGroup;
 
 public class PaymentButtonsFragment extends BaseFragment {
 
     private PayPalButton payPalButton;
-    private MaterialButtonToggleGroup toggleGroup;
+    private MaterialButtonToggleGroup ppToggleGroup;
```

Comment:
Small nit to maybe rename this to `payPalToggleGroup` to match the venmo naming.